modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,106 @@ payload
 
 // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// Flat File config properties are shared with COBOL Copybook and Fixed Width.
-include::partial$dataweave-formats-flatfile-config.adoc[]
+
```

Comment:
TODO DEV: can we output the config properties to a separate file (dataweave-formats-flatfile-config.adoc) in the _partials subdirectory for flat file properties?